Modules/Loadable/Segmentations/Widgets/qMRMLSegmentEditorWidget.cxx

```diff
@@ -328,15 +328,16 @@ qMRMLSegmentEditorWidgetPrivate::qMRMLSegmentEditorWidgetPrivate(qMRMLSegmentEdi
   this->EffectNameOrder
     // Thresholding is the the starting point for most segmentations
     // (it can often create usable segmentation by itself, or used to define intensity range for painting)
-    << "Threshold"
+    << qMRMLSegmentEditorWidget::tr("Threshold")
```

Comment:
These are effect names here (not effect titles) therefore they must not be translated